### ISSUE TITLE
[UNDER-DISCUSSION] The Plugin install script can only be executed from the RAVEN folder

### ISSUE THREAD

**Issue by wangcj05** [idaholab/raven#1112]
--------
Under Discussion Topic
--------

**Summary of the topic to be discussed with the development team**
A clear and concise description of the topic to be discusses

```
(raven_libraries) [wangc][~/projects/tests/raven/scripts]> python install_plugins.py -a
Initializing plugin "PRAplugin"
 ... initializing plugin submodule PRAplugin ...
error: pathspec 'plugins/PRAplugin' did not match any file(s) known to git
 ... plugin located at "/Users/wangc/projects/tests/raven/plugins/PRAplugin" ...
Initializing plugin "ExamplePlugin"
 ... initializing plugin submodule ExamplePlugin ...
error: pathspec 'plugins/ExamplePlugin' did not match any file(s) known to git
 ... plugin located at "/Users/wangc/projects/tests/raven/plugins/ExamplePlugin" ...
Installing plugins ...
 ... plugin "PRAplugin" path updated to "/Users/wangc/projects/tests/raven/plugins/PRAplugin" ...
 ... plugin "PRAplugin" informed of RAVEN at "/Users/wangc/projects/tests/raven/framework"...
 ... plugin "PRAplugin" succesfully installed!
 ... plugin "ExamplePlugin" path updated to "/Users/wangc/projects/tests/raven/plugins/ExamplePlugin" ...
 ... plugin "ExamplePlugin" informed of RAVEN at "/Users/wangc/projects/tests/raven/framework"...
 ... plugin "ExamplePlugin" succesfully installed!
(raven_libraries) [wangc][~/projects/tests/raven/scripts]> cd ../
(raven_libraries) [wangc][~/projects/tests/raven]> python scripts/install_plugins.py -a
Initializing plugin "CashFlow"
 ... initializing plugin submodule CashFlow ...
 ... plugin located at "/Users/wangc/projects/tests/raven/plugins/CashFlow" ...
Initializing plugin "PRAplugin"
 ... initializing plugin submodule PRAplugin ...
 ... plugin located at "/Users/wangc/projects/tests/raven/plugins/PRAplugin" ...
Initializing plugin "ExamplePlugin"
 ... initializing plugin submodule ExamplePlugin ...
 ... plugin located at "/Users/wangc/projects/tests/raven/plugins/ExamplePlugin" ...
Installing plugins ...
 ... plugin "CashFlow" path updated to "/Users/wangc/projects/tests/raven/plugins/CashFlow" ...
 ... plugin "CashFlow" informed of RAVEN at "/Users/wangc/projects/tests/raven/framework"...
 ... plugin "CashFlow" succesfully installed!
 ... plugin "PRAplugin" path updated to "/Users/wangc/projects/tests/raven/plugins/PRAplugin" ...
 ... plugin "PRAplugin" informed of RAVEN at "/Users/wangc/projects/tests/raven/framework"...
 ... plugin "PRAplugin" succesfully installed!
 ... plugin "ExamplePlugin" path updated to "/Users/wangc/projects/tests/raven/plugins/ExamplePlugin" ...
 ... plugin "ExamplePlugin" informed of RAVEN at "/Users/wangc/projects/tests/raven/framework"...
 ... plugin "ExamplePlugin" succesfully installed!
(raven_libraries) [wangc][~/projects/tests/raven]>
```

**Describe the solution you'd like to be implemented**

**Describe alternatives you've considered**

----------------
For Change Control Board: Issue Review
----------------
This review should occur before any development is performed as a response to this issue.
- [x] 1. Is it tagged with the under_discussion type?
- [x] 2. If implemented, it will add a new requirement? __no new requirement__
- [x] 3. Is a rationale provided? (Such as explaining why the improvement is needed )

-------
For Change Control Board: Issue Closure
-------
This review should occur when the issue is imminently going to be closed.
- [x] 1. The discussion determined the addition of a new ```task``` issue?

**Comment by PaulTalbot-INL** [comment]
Ah, this is because we used "git config" from an `os.popen` call without setting the working dir. This should be an easy fix.

**Comment by wangcj05** [comment]
This is fixe by #1115, and no additional issue is required. This issue can be closed when PR #1115 is merged.
